root.sh

```diff
@@ -103,8 +103,74 @@ else
 fi
 
 unset DYLD_LIBRARY_PATH
+
+mkdir -p ./src_tmp
```

Comment:
This means that  ROOT development package will stop working. Can we merge the patch in our own fork?

Reply:
I was copy and pasting how other recipes were modifying the source code. Is there a better way?
What do you mean by "merge the patch in our own fork"?

Reply:
Yeah, but for ROOT we should keep the development packages functional. This will break things because the rsync decouples the local checkout from the one used for building. Can you do a PR on top of https://github.com/alisw/root/tree/alice/v6-18-04?

Reply:
It requires 2 patches, the first is in master but no in 6.18.4-patches, and the second is not even merged in master. We can of course create an 6-18-04-alice tag in our fork, and use that. Then we can do away with the life-patching.
But I cannot write to the alisw fork. Or do you want me to do it in my own fork for the time being?

Reply:
I can give you access, one sec.

Reply:
can you try now? you should have write access to alisw/root

Reply:
ok, worked, tag is there, I am creating a PR to alidist

Reply:
I have updated #1918 with the new root recipe that does not do life patching. But we have to wait for the CI again.

Reply:
Ok, thanks. I will merge it as soon as it passes.